FIX postgres-compatibility unskipped tests

postgres-compatibility  - +236 тестов (проблема с docker, были заскипаны, но что-то уровне заскипов его тестов сломалось и они начали запускаться ) - https://nda.ya.ru/t/bcOKueWa7Egftu

будет оч круто если в ближайшее время у тебя получится посмотреть/ починить - они все попадают в retry в CI 

Вот пример отчета 
https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/PR-check/15165942929/ya-x86-64/try_1/ya-test.html#MUTE

Вот ошибка

```
setup failed:
ydb/tests/postgres_integrations/go-libpq/docker_wrapper_test.py:15: in setup_module
    tl.setup_module(module)
ydb/tests/postgres_integrations/library/pytest_integration.py:92: in setup_module
    image = _docker_build(_tests_folder)
ydb/tests/postgres_integrations/library/pytest_integration.py:150: in _docker_build
    client.images.build(
contrib/python/docker/docker/models/images.py:304: in build
    raise BuildError(chunk['error'], result_stream)
E   docker.errors.BuildError: toomanyrequests: You have reached your unauthenticated pull rate limit. https://www.docker.com/increase-rate-limit```

--- a/ydb/tests/postgres_integrations/go-libpq/data/Dockerfile
+++ b/ydb/tests/postgres_integrations/go-libpq/data/Dockerfile
@@ -1,6 +1,6 @@
 # For docker context at root git directory
 
-FROM cr.yandex/crpq25pvu31gqulohlbv/golang:1.20-amd64
+FROM cr.yandex/crp9h1qbla5olt8ugd4m/golang:1.20-amd64
 
 WORKDIR /project/sources/
 

--- a/ydb/tests/postgres_integrations/go-libpq/data/Dockerfile
+++ b/ydb/tests/postgres_integrations/go-libpq/data/Dockerfile
@@ -1,6 +1,6 @@
 # For docker context at root git directory
 
-FROM golang:1.20
+FROM ghcr.io/ydb-platform/golang:1.20
 
 WORKDIR /project/sources/
 

--- a/ydb/tests/postgres_integrations/go-libpq/data/Dockerfile
+++ b/ydb/tests/postgres_integrations/go-libpq/data/Dockerfile
@@ -1,6 +1,6 @@
 # For docker context at root git directory
 
-FROM ghcr.io/ydb-platform/golang:1.20-amd64
+FROM cr.yandex/crpq25pvu31gqulohlbv/golang:1.20-amd64
 
 WORKDIR /project/sources/
 

--- a/ydb/tests/postgres_integrations/go-libpq/data/Dockerfile
+++ b/ydb/tests/postgres_integrations/go-libpq/data/Dockerfile
@@ -1,6 +1,6 @@
 # For docker context at root git directory
 
-FROM ghcr.io/ydb-platform/golang:1.20
+FROM ghcr.io/ydb-platform/golang:1.20-amd64
 
 WORKDIR /project/sources/
 

